### PR TITLE
Manifest json ignoring

### PR DESCRIPTION
**Pull request by joeycozza**
Using a variable provided from snow to determine if we should add the manifest.json file to layout.ejs.

The `typeof manifestJsonExists === 'undefined'` check will make sure that if someone gets the @fs/react-scripts update without getting the snow update, they will continue to provide the manifest.json and the page won't blow up.

### CODE REVIEW

File: packages/react-scripts/layout/views/layout.ejs [hunk 1]
```diff
@@ -15,7 +15,10 @@
     <meta name="google-site-verification" content="VJlFQrAXTCBy0PqVi23EsGD-mpA5KaBhIt8LkrcRbP0" />
     <link href="https://edge.fscdn.org/assets/docs/fs_logo_favicon_sq.png" rel="icon" type="image/x-icon" />
     <link rel="apple-touch-icon" href="https://edge.fscdn.org/assets/components/hf/assets/img/tree-touch-icon-11024a277f1fda5735de5a9f0f4f75ca.png" />
-    <link rel="manifest" href="<%= typeof appPathOverride !== 'undefined' ? appPathOverride : appPath('') %>/manifest.json" type="application/manifest+json" crossorigin="use-credentials" />
+
+    <% if (manifestJsonExists) { %>
```
Comment (johndaley-me) [inline packages/react-scripts/layout/views/layout.ejs]: This looks good. Your pull request description though mentioned `typeof manifestJsonExists === 'undefined'`. Is that something you are going to change this to or did you push your latest changes?

Comment (joeycozza) [reply packages/react-scripts/layout/views/layout.ejs]: Thanks, I did that in my testing app and forgot to put it here.

**Review by johndaley-me — APPROVED** [review]
🥇

**Review by thorntonjn — APPROVED** [review]
Thank you!

**Pull request merged**